### ISSUE TITLE
Contradiction with the rustc book on --cap-lints and forbid

### ISSUE THREAD

**Issue by josephcsible** [rust-unofficial/patterns#281]
https://github.com/rust-unofficial/patterns/blob/66d7e6c7daea2b6cf50f6cb230c7203ece0d8678/anti_patterns/deny-warnings.md#L39-L42

This seems to contradict this:

https://github.com/rust-lang/rust/blob/3bf55753c680e653eaa9d11dd66e56844bd04d32/src/doc/rustc/src/lints/levels.md#L102-L105

> However, lint levels may still be capped with `--cap-lints` (see below) so `rustc --cap-lints warn` will make lints set to 'forbid' just warn.

**Comment by MarcoIeni** [comment]
Thanks, I have done a PR to fix this. Can you review #282?

**Comment by MarcoIeni** [comment]
Yes, I know, don't worry, I just wanted to make sure you liked my change :)
Let's wait for simonsan
